packages/admin/src/pages/build-application/[applicationId]/[sectionId]/question-options.page.tsx

```diff
@@ -61,36 +91,57 @@ export const getServerSideProps: GetServerSideProps = async ({
     res,
     async (body: any) => {
       options = body.options;
+      const { optional, ...restOfQuestionSummary } = questionSummary;
 
       if ('add-another-option' in body) {
         options.push('');
 
         return {
-          data: 'OPTION_ADDED',
+          data: '',
         };
-      } else {
-        const { optional, ...restOfQuestionSummary } = questionSummary;
+      } else if (body.questionId) {
+        await patchQuestion(
+          sessionId,
+          applicationId,
+          sectionId,
+          body.questionId,
+          {
+            ...restOfQuestionSummary,
+            options,
+            validation: {
+              mandatory: optional !== 'true',
+              maxWords: '',
+            },
+          }
+        );
 
+        const editQueryString = query.backTo
+          ? `?${new URLSearchParams({ backTo: query.backTo.toString() })}`
```

Comment:
template literal makes this a little harder to read

```suggestion
          ? '?' + new URLSearchParams({ backTo: query.backTo.toString() })
```